functorch/einops/rearrange.py

```diff
@@ -20,8 +20,6 @@
 
 __all__ = ["rearrange"]
 
-dims = _C.dims
```

Comment:
We should mark this as BC-breaking. From my chat with Richard, he was ok with us removing the old code here but we should still mention it on the release notes.

Reply:
It's... not BC breaking though? I import dims from functorch.dim so it's still available in this namespace. Unless you're saying that a module import has different assignments than assigning it?